build.gradle.kts

```diff
@@ -1,11 +1,11 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 plugins {
-    kotlin("jvm") version Versions.kotlin
+    kotlin("jvm") version "1.4.0"
```

Comment:
So is the issue here that this is "too early" to grab the version directly?

Reply:
Exactly, that's my understanding. I ended up bringing back the `Versions.kt` just for those, since it still seems a bit cleaner than directly specifying them, but most (actual) dependencies can now go in `libs.versions.toml`.

Reply:
Gotcha, makes sense.